docs/UserGuide.adoc

```diff
@@ -53,7 +53,81 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 === Recipe inventory commands
 
-=== Database commands
+=== Cookbook commands
+==== View a recipe from the cookbook
+View a recipe, its ingredients, and the steps to cook it.
+
+- Format: `cookbook view recipe INDEX`
+- Example:
+* `cookbook view recipe 1`
+
+==== Tag a recipe
+Tag a recipe.
+
+- Format: `cookbook tag recipe INDEX t/TAG…`
+- Examples:
+* `cookbook tag recipe 1 t/Carbonara`
+* `cookbook tag recipe 1 t/Carbonara t/Pasta t/Cream`
+
+==== Untag a recipe
+Untag a recipe.
+
+[TIP]
+If no tags are specified, all the tags for the recipe will be removed.
+
+- Format: `cookbook untag recipe INDEX [t/TAG]…`
+- Examples:
+* `recipe untag 1`
+* `recipe untag 1 t/Carbonara`
+
+==== Search recipes by keyword
+Search for a recipe by a keyword, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: `cookbook search recipe w/KEYWORD`
+- Example:
+* `cookbook search recipe w/Carbonara`
+
+==== Search recipes by tag
+Search for recipes by tags, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: cookbook search tag [t/TAG]…
```

Comment:
The markup is missing. Do we want to require at least one tag? What is the intended behaviour when the user doesn't provide any tags?

Reply:
What do you mean by markup? Anyway if the user doesn't provide a tag, it should be fine. I don't think we should make tags mandatory.

Reply:
as in Format: `command`

so that means we will display all recipes if no tag is given?

Reply:
right I get what you mean.  added the markup.

in that case I'll change it to ```cookbook search tag [t/TAG]...``` so a minimum of a tag is required for search. a search without indicating 1 tag will return an error message?

Reply:
sounds good